src/misc/news/NewsListItem.ts

```diff
@@ -0,0 +1,27 @@
+import m, {Children} from "mithril"
+import {NewsId} from "../../api/entities/tutanota/TypeRefs.js"
+import {NewsModel} from "./NewsModel.js"
+
+export abstract class NewsListItem {
```

Comment:
I am not sure about this whole thing tbh.

1. It mixes up the logic and presentation which makes it tricky to test things properly
2. Is is unnecessary class hierarchy (`acknowledge()` should be a method on newsModel, why is it here?)

I would propose the following:
1. Make `NewsItem` a model object. The interface can have `isShown()`
2. Introduce view components for news items.
```ts
interface NewsItemComponent<T extends NewsItem> extends Component<{item: T}>
```
3. Model only operates on NewsItem's. They have whatever logic and dependencies they need. Since we have no proper injection we need to pass deps to them for now. Instantiating them can be a `switch` statement or whatever.
4. `NewsItemList` gets a list off models and maps them to view components. Yup, I know, that's another `switch` but this is cleaner.

Reply:
We decided to not split for now